en/vespa8-release-notes.html

```diff
@@ -873,6 +874,39 @@ <h3 id="vespa-http-client">vespa-http-client</h3>
 </p>
 
 
+<h2 id="renamed-metrics">Renamed metrics</h2>
+<p>
+The following metrics are renamed:
+</p>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Old Name</th><th>Name name</th><th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><em>vds.filestor.alldisks.*</em></td>
+      <td>vds.filestor.*</td>
+      <td><em>alldisks</em> has been removed from the metric name.</td>
+    </tr>
+    <tr>
+      <td><em>vds.visitor.*.sum.*</em></td>
+      <td>vds.visitor.*.*</td>
+      <td><em>sum</em> has been removed from the metric name.</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.*.sum.*</em></td>
+      <td>vds.filestor.*.*</td>
+      <td><em>sum</em> has been removed from the metric name.</td>
+    </tr>
+    <tr>
+      <td><em>vds.distribtor.*.sum.*</em></td>
+      <td>vds.distribtor.*.*</td>
```

Comment:
s/distribtor/distributor for these two lines

Reply:
Fixed